test/system/000-setup.bats

```diff
@@ -15,6 +15,12 @@
 # limitations under the License.
 #
 
+if [ ! -f test/system/libs/helpers.bash ]; then
```

Comment:
`test/system/libs/helpers.bash` doesn't come from the `bats-assert` and `bats-support` submodules - it's part of Toolbx itself.  So, I think this check needs to look for things like `test/system/libs/bats-assert/load.bash` and such.

Reply:
Done.  Changed to `test/system/libs/bats-assert/load.bash` and `test/system/libs/bats-support/load.bash`

Reply:
Also, it should be `if ! [ -f ...`, not `if [ ! -f ...`, because `[` is actually a shell command that takes `-f ...` as its arguments and we need to negate the whole command.  Yes, shell scripts are weird.  :)